TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/config/SecurityConfig.java

```diff
@@ -24,7 +24,9 @@ public class SecurityConfig {
     private static final String[] WHITE_LIST_URL = {
             "/auth/**",
             "/attractions/getAll",
-            "/reviews/getAllByAttractionId/*"
+            "/reviews/getAllByAttractionId/*",
+            "/websocket/**"
```

Comment:
W: Theoretically, the web-socket URL should not be whitelisted, but that is outside of our scope now